src/teleop/gui/src/conf.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Seems a little strange to include these files in gui/src/. Are they needed for what this PR does?